static/js/app.js

```diff
@@ -487,6 +487,33 @@ $('.js-validate-filters').each(function() {
 });
 
 /* ==========================================================================
+   Pagination Form validation
+
+   Check to verify page input value is number and with min/max range
+   before submitting form.
+   ========================================================================== */
+
+$('.js-validate_pagination').each(function() {
```

Comment:
Let's add white space to wrapping parens to match what we've been doing elsewhere.